gp-includes/route.php

```diff
@@ -168,8 +168,8 @@ public function invalid_nonce_and_redirect( $action, $url = null ) {
 	 *                                 Default: 'You are not allowed to do that!'.
 	 * @return false
 	 */
-	public function can_or_forbidden( $action, $object_type = null, $object_id = null, $message = 'You are not allowed to do that!' ) {
-		$can = $this->can( $action, $object_type, $object_id );
+	public function can_or_forbidden( $action, $object_type = null, $object_id = null, $extra = null, $message = 'You are not allowed to do that!' ) {
```

Comment:
You're right. I checked the source and the message is currently no use of the `$message` parameter. If I put the `$extra` parameter last then I'd need to duplicate the message string here: https://github.com/akirk/GlotPress/blob/add/cross-pte-filters/gp-includes/routes/translation.php#L603

Any suggestions on how to solve this?